blue_bringup/launch/full.launch

```diff
@@ -1,6 +1,7 @@
 <launch>
   <arg name="param_file" />
   <arg name="version" default="2" doc="1 or 2" />
+  <arg name="on_stand" value="false" /> <!-- this value should not matter -->
```

Comment:
I know it doesn't actually do anything, but might it make more sense to default this to `true`?